mrtt-ui/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended'
```

Comment:
We can add to this as needed later. `react-app/jest` should probably added when we add testing libraries.

Reply:
will add!